tests/unittests/qml/tst_PreviewManager.qml

```diff
@@ -69,7 +70,9 @@ Item {
                 HistoryModel.add(url, "Example Com" + i, "")
                 if (createPreviewFiles) {
                     var path = PreviewManager.previewPathFromUrl(url)
+                    console.log("the preview path " + path + " for url " + url + " is created.")
```

Comment:
Accidentally left in console.log?

Reply:
I guess so, it was for debugging -> I'll remove it